### PR TITLE
feat: Implement Endpoints for Language Management

### PR DESCRIPTION
**Pull request by dorisjenny27**
# Description

**Closes #issue_number_here**

# Changes proposed

## What were you told to do?
Implement endpoint to create language
Implement endpoint to get languages by Id
Implement endpoint to update language
Implement endpoint to get all languages
Implement endpoint to delete languages 

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **dev branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

File: src/Hng.Application/Features/Languages/Dtos/LanguageResponseDto.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Hng.Application.Features.Languages.Dtos
+{
+    public class LanguageResponseDto
+    {
+        [JsonPropertyName("language")]
+        public LanguageDto Language { get; set; }
```
Comment (Godhanded) [inline src/Hng.Application/Features/Languages/Dtos/LanguageResponseDto.cs:8]: Please rebane this the data field, so as to follow convention

File: src/Hng.Application/Features/Languages/Dtos/LanguagesListResponseDto.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Hng.Application.Features.Languages.Dtos
+{
+    public class LanguagesListResponseDto
+    {
+        [JsonPropertyName("languages")]
+        public List<LanguageDto> Languages { get; set; }
```
Comment (Godhanded) [inline src/Hng.Application/Features/Languages/Dtos/LanguagesListResponseDto.cs:13]: Rename this field to data so as to follow convention

File: src/Hng.Web/appsettings.json [hunk 3]
```diff
@@ -36,9 +36,9 @@
     "ApiSecret": ""
   },
   "EmailTemplateDirectory": {
-    "path": ""
+    "path": "../Hng.Infrastructure/EmailTemplates"
```
Comment (Godhanded) [inline src/Hng.Web/appsettings.json:39]: Remove appsettings secretes

**Review by Godhanded — CHANGES_REQUESTED** [review]

File: src/Hng.Web/appsettings.json (hunk 3 above)
Comment (dorisjenny27) [reply src/Hng.Web/appsettings.json:39]: I pulled this appsettings this dev. Will adjust it anyway but ask the person that initially pushed this to make necessary adjustments on his part too.

File: src/Hng.Application/Features/Languages/Dtos/LanguageResponseDto.cs (hunk 1 above)
Comment (dorisjenny27) [reply src/Hng.Application/Features/Languages/Dtos/LanguageResponseDto.cs:8]: I do not understand what you mean here.

File: src/Hng.Application/Features/Languages/Dtos/LanguagesListResponseDto.cs (hunk 2 above)
Comment (dorisjenny27) [reply src/Hng.Application/Features/Languages/Dtos/LanguagesListResponseDto.cs:13]: I do not understand what you mean here as well

**Review by chimaihueze — APPROVED** [review]

**Pull request merged**